src/lambda_function.py

```diff
@@ -195,7 +195,6 @@ def test_connection_to_community(community):
 
 
 def lambda_handler(event, context):
-
     communities = get_dmk_community_list_from_cloud()
     for community in communities:
         success = test_connection_to_community(community)
```

Comment:
Hey...so...for the whole function you wrote there and the corresponding s3 stuff and the soap stuff...wouldn't it have been just easier to call this endpoint on our cloud?

/v3/integrations/dormakaba/test/

Cuz that should also be able to verify if a server is responding or not. And even if that endpoint itself doesn't necessarily work - wouldn't it be better to just write an endpoint on the cloud side that sent that soap message? Cuz like we have all the functions already there in cloud and this lambda and the common lambda lib kinda just duplicates a lott of it.

Reply:
Just to keep the footprint of this lambda as small and as simple as possible